src/App.tsx

```diff
@@ -10,33 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
```

Comment:
Let's get used to name functions according to these rules:
![image](https://user-images.githubusercontent.com/96409563/208483489-90589187-c988-4c2f-9c7b-3f1d6cf66991.png)

```suggestion
  const selectTab = (tab: Tab) => {
```